build.sbt

```diff
@@ -124,7 +104,6 @@ lazy val giapi = project
   .enablePlugins(GitBranchPrompt)
   .settings(commonSettings: _*)
   .settings(
-    addCompilerPlugin(Plugins.kindProjectorPlugin),
```

Comment:
Why don't we need this anymore?

Reply:
You get it for free from sbt-typelevel-settings.